src/main/java/DudeCommand.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Commands to be entered into Dude.
+ */
+public enum DudeCommand {
+  list,
```

Comment:
I think enums should be capitalized since they are like constants. What do you think?